### ISSUE TITLE
Issue with params and http client

### ISSUE THREAD

**Issue by FiV0** [xtdb/crux#2933]
The following works locally but not against the http client. I suppose some issue with parameters.
```clj
(xt/submit-tx node [[:put :docs {:xt/id :petr :name "Petr"}]])

(xt/q node '(from :docs [{:xt/id $petr} name]) {:args {:petr :petr}})
```

**Comment by FiV0** [comment]
This is due to request coercion and that currently XTQL doesn't support the vectorized `[q args]` syntax.

**Comment by FiV0** [comment]
Fixed in cbf1ef73
